cmd/client-fs.go

```diff
@@ -223,29 +223,42 @@ func (f *fsClient) Watch(ctx context.Context, options WatchOptions) (*WatchObjec
 }
 
 func preserveAttributes(fd *os.File, attr map[string]string) *probe.Error {
-	mode, e := strconv.ParseUint(attr["mode"], 10, 32)
-	if e != nil {
-		return probe.NewError(e)
-	}
+	if val, ok := attr["mode"]; ok {
+		mode, e := strconv.ParseUint(val, 10, 32)
```

Comment:
Since the mode is usually expressed in base 8 as `0644` or `0777` ParseUint should specify 0 as the base, to allow interpretation based on the prefix:
```suggestion
		mode, e := strconv.ParseUint(val, 0, 32)
```

Otherwise I think there is a bug: `0777` interpreted as base 10 is not the same as `0777` interpreted as base  8.

Reply:
@donatello 
Great catch? This is the old code and already existing bug.
Fixed it.